uptane/services/inventorydb.py

```diff
@@ -169,4 +84,122 @@ def scrub_filename(fname, expected_containing_dir):
   return abs_fname
 
 
+class InventoryDataBase():
+  """
+  Dictionary design:
+
+  {
+    'vehicle manifests':
```

Comment:
After looking through the code, I think I would probably split this into three or four separate global (module-level) variables: vehicle_manifests, ecu_manifests, public_keys (or perhaps primary_public_keys and all_public_keys instead of public_keys).